src/core/server/cron/accountDeletion.ts

```diff
@@ -0,0 +1,223 @@
+import { CronJob } from "cron";
+import { DateTime } from "luxon";
+import { Collection, Db } from "mongodb";
+
+import logger from "coral-server/logger";
+import { createCollection } from "coral-server/models/helpers";
+import { Story } from "coral-server/models/story";
+import { Tenant } from "coral-server/models/tenant";
+import { User } from "coral-server/models/user";
+import { MailerQueue } from "coral-server/queue/tasks/mailer";
+
+const BATCH_SIZE = 500;
+
+export function registerAccountDeletion(
+  mongo: Db,
+  mailer: MailerQueue
+): CronJob {
+  const job = new CronJob({
+    cronTime: "0,30 * * * *",
+    timeZone: "America/New_York",
+    start: true,
+    runOnInit: false,
+    onTick: async () => {
+      deleteScheduledAccounts(mongo, mailer);
```

Comment:
Instead, turn `deleteScheduledAccounts` into a function that receives the mongo and mailer references and returns a `CronCommand`.

```js
function deleteScheduledAccounts(mongo: Db, mailer: MailerQueue): CronCommand {
  return async () => {
```